wemake_python_styleguide/visitors/tokenize/comments.py

```diff
@@ -191,15 +191,26 @@ def _check_empty_line_after_codding(
 class ShebangVisitor(BaseTokenVisitor):
     """Checks the first shebang in the file."""
 
+    _shebang: ClassVar[Pattern] = re.compile(r'(\s*)#!')
+    _python_executable: ClassVar[str] = 'python'
+    _comments_or_newlines = NEWLINES.union({tokenize.COMMENT})
+    _violation_token = tokenize.TokenInfo(
```

Comment:
We don't need this token. Instead, we can inherit `ShebangViolation` from `SimpleViolation`